modules/runner/src/integrationTest/sql/subquery/any_all/test_uncorrelated_all_subquery.test_ignore

```diff
@@ -1,217 +0,0 @@
-# name: test/sql/subquery/any_all/test_uncorrelated_all_subquery.test
-# description: Test uncorrelated ALL subqueries
-# group: [any_all]
-# Ignore https://issues.apache.org/jira/browse/IGNITE-15583
```

Comment:
the same: we need to adjust issue as well, not only change the code

Reply:
yes, but after merge the PR